src/components/organisms/plan/plan.scss

```
@@ -0,0 +1,39 @@
+@use "../../variables" as *;
+
+.mol-plan{
+  margin-top: 56px;
+  .modMessage{
+    padding-top: 0;
+    padding-bottom: 56px;
+    h2{
+      font-size: $fontsize-l;
+      color: $orange_bistre;
+      margin-bottom: 6px;
+    }
+    p{
+      font-size: $fontsize-s;
+    }
+  }
+  .plan-assets{
+    .plan{
+      margin-bottom: 24px;
+      @media (min-width:$breakpoint-md) {
+        margin-bottom: 40px;
+        margin-right: 30px;
+        &:nth-child(3){
+          margin-right: 0;
+        }
+      }
+    }
+    @media (min-width:$breakpoint-md) {
+      display: flex;
```

```suggestion
      }
    }
    @media (min-width:$breakpoint-md) {
      display: flex;
      gap: 30px;
```
`gap` プロパティを使うとよりスッキリと書けると思います！
- nth-childを使わなくてもいい
- 親要素の方に配置関連の記述をまとめることができる

あたりがメリットです